test/acceptance/framework/k8s/debug.go

```diff
@@ -78,20 +78,29 @@ func WritePodsDebugInfoIfFailed(t *testing.T, kubectlOptions *k8s.KubectlOptions
 
 		// Describe any stateful sets.
 		statefulSets, err := client.AppsV1().StatefulSets(kubectlOptions.Namespace).List(context.Background(), metav1.ListOptions{LabelSelector: labelSelector})
+		if err != nil {
+			logger.Log(t, "unable to get statefulsets", "err", err)
+		}
 		for _, statefulSet := range statefulSets.Items {
 			// Describe stateful set and write it to a file.
 			writeResourceInfoToFile(t, statefulSet.Name, "statefulset", testDebugDirectory, kubectlOptions)
 		}
```

Comment:
```suggestion
		if err != nil {
			logger.Log(t, "unable to get statefulsets", "err", err)
		}
		else {
			for _, statefulSet := range statefulSets.Items {
				// Describe stateful set and write it to a file.
				writeResourceInfoToFile(t, statefulSet.Name, "statefulset", testDebugDirectory, kubectlOptions)
			}
		}
```

I think we need an `else` for all of these cuz the resource could be `nil` and then `.Items` will panic.